src/harness/client.ts

```diff
@@ -568,12 +568,13 @@ export class SessionClient implements LanguageService {
         return notImplemented();
     }
 
-    findRenameLocations(fileName: string, position: number, findInStrings: boolean, findInComments: boolean, providePrefixAndSuffixTextForRename?: boolean): RenameLocation[] {
+    findRenameLocations(fileName: string, position: number, findInStrings: boolean, findInComments: boolean, preferences: UserPreferences): RenameLocation[] {
         if (!this.lastRenameEntry ||
             this.lastRenameEntry.inputs.fileName !== fileName ||
             this.lastRenameEntry.inputs.position !== position ||
             this.lastRenameEntry.inputs.findInStrings !== findInStrings ||
             this.lastRenameEntry.inputs.findInComments !== findInComments) {
+            const { providePrefixAndSuffixTextForRename } = preferences;
             if (providePrefixAndSuffixTextForRename !== undefined) {

```

Comment:
Not sure if I should also have a check for `quotePreference` here. IIRC the harness is for testing purposes so I thought it didn't matter (?)

Reply:
You’ll need it if you want quote preference settings to be testable in fourslash server tests (I think that’s what this is used for, it’s hard to keep track)